deeplinkdispatch-processor/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkAnnotatedElement.java

```diff
@@ -52,8 +52,14 @@ String getUri() {
     return annotationType;
   }
 
-  TypeElement getActivityElement() {
-    return activityElement;
+  TypeElement getAnnotatedElement() {
+    return annotatedElement;
+  }
+
+  /** Returns the package name for the annotated TypeElement. */
+  String getPackageName() {
+    String qualifiedName = annotatedElement.getQualifiedName().toString();
```

Comment:
`javax.lang.model.util.Elements#getPackageOf` will this work instead of string parsing? I think your existing code may break with inner classes